src/objects/Levels_Settings__c.object

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@ptewson-sfdo I'm not sure it's worth adding a new custom object for this single field. If anything I would recommend a BatchSettings object so we can add other batch job size settings going forward. However, given that all the Rollup batch job size settings are in the CustomizableRollups object and some of the other batch job size settings are in other settings objects, I would suggest this just be added as a field to the Contacts & Organizations or Households custom settings object. It would eliminate a lot of the code and metadata/

Reply:
@force2b We discussed this and it sounds like there is potential for additional Levels settings, so decided to add this now even if it's a bit thin.

Reply:
Oh. What other level settings are you thinking of? I don't really see anything in the backlog for Levels. If we do want to keep that object then the PR will need the translation files.

Reply:
We were thinking of a setting for filtering records included in the batch - see W-030738. 

Good call on translations!

Reply:
If that’s the wi Jess just added today, there are no settings needed on that wi, at least we didn’t identify any in refinement.

Reply:
Just to close the loop from the WI posting, let's keep with this object. I will update the other WI to reference this.

Reply:
Thanks for the hot tip on translations! Will include that asap.

Reply:
@ptewson-sfdo Holding my final approval until the translation files are in to avoid the build failures in master. CC @rponti-sforg

Reply:
@force2b @rponti-sforg - are we looking for more translation files than the files like "Levels_Settings__c-nl_NL.objectTranslation" that are already included?